BaSP-a2022/ayom-a-trackgenix-app
# BaSP-TG-144: Edit/Create admin (SuperAdmin)

- Create form to create/edit admins
- Once created/edited redirect to the super admin home

QA: 
How to test?:
How to test?:
Log in as a Super Admin (email: [oclacep@wikimedia.org](mailto:oclacep@wikimedia.org), password: prueba1234)
When logged you should se a nav bar (temporary), move to the first "form" link (Add admin form)
You should see a form with empty inputs.
The "cancel" button must redirect you to the page you were before.
The "reset" button shuod clear the fields if you wrote something on them.
Once completed the fields the "confirm" button should open a modal to confirm the creation of a new admin, On cancel, mdoal will close. On confirmyou should create the new user, and a success modal will appear.
Move to home link again.
Now move to the second "form" link (Update admin form).
You should see a form with filled inputs (this is a hardcoded id, so the fields will be always the same)
The cancel and reset buttons should work as in the add admin form.
To test the "confirm" button change some of the fields (don't change the email, and you NEED to change the password (please change the password to "prueba1234" once you've finished with the tests).
Note: The nav bar is temporaryand just to use on the test of the PRs, any bad function on navigation should be ignored.
